README.md

```diff
@@ -64,13 +60,47 @@ For the full reference of possible configuration values, see the [config schema
     make help
     ```
 
-    > Note: If you're deploying for the first time and not using `make all` (i.e. using `make deploy-core`), ensure you have ran `make bootstrap` first.
-
 ### CI (GitHub Actions)
 
 CI deployment workflows are run in [Github environments](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment). These should
 be created in a private repository created from this template repository.
 
-<!-- 
-very much a work in progress here...
--->
+1. Create a service principal
+
+    CI deployments require a service principal with access to deploy resources
+    in the subscription. One will be required for each subscription into which the
+    environment deploys. Create one with:
+
+    ```bash
+    subscription_id=<e.g 00000000-0000-0000-0000-00000000>
+    az ad sp create-for-rbac --name "sp-flowehr-cicd" --role Owner --scopes "/subscriptions/${subscription_id}"
+    ```
+
+    The output will be used in the next step.
+
+
+2. Create and populate a GitHub environment
+
+    Add an environment called `Infra-Test` with following secrets
+
+    - `AZURE_CREDENTIALS`: json containing the credentials of the service principal in the format
+
+    ```json
+    {
+    "clientId": "xxx",
+    "clientSecret": "xxx",
+    "tenantId": "xxx",
+    "subscriptionId": "xxx",
+    "resourceManagerEndpointUrl": "management.azure.com"
+    }
+    ```
+
+    - `PREFIX`: Prefix used for naming resources. Must be unique to this repository e.g. `abcd`
+    - `LOCATION`: Name of an Azure location e.g. `uksouth`. These can be listed with `az account list-locations -o table`
+    - `ENVIRONMENT`: Name of the environment e.g. `dev`, also used to name resources
+    - `DEVCONTAINER_ACR_NAME`: Name of the azure container registry to use for the devcontainer build. This may or may not exist. e.g. `flowehrmgmtacr`
```

Comment:
```suggestion
    - `DEVCONTAINER_ACR_NAME`: Name of the Azure Container Registry to use for the devcontainer build. This may or may not exist. e.g. `flowehrmgmtacr`
```